app/controllers/api/v1/profiles_controller.rb

```diff
@@ -31,6 +31,12 @@ def set_active_account
     head :ok
   end
 
+  def resend_confirmation
+    return @user.send_confirmation_instructions if @user
```

Comment:
Should we check if the user is confirmed already?

Reply:
Yep we can, I'll update the PR